app/assets/stylesheets/modules/forms.scss

```diff
@@ -83,6 +83,12 @@ hr {
   display: none;
 }
 
+// This is done because the html5 shim removes elements
+// with the .help-block class after selecting a date
```

Comment:
thanks for this comment!   I'm not clear enough on "html5 shim removes elements" meaning tho - does the .help-block class remove particular things in the dom?   This is just generic enough to make me wonder more about the problem this is fixing.  Is there a reference that could be supplied?   Or am I being too picky?